deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -126,6 +126,11 @@ spec:
                     items:
                       type: string
                       pattern: '^/.*[^/]$'
+                  eventList:
+                    type: array
+                    items:
+                      type: string
+                      pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
```

Comment:
Instead of specifying a pattern here we should enumerate the possible values. As an example of doing that, cf. the serviceType property in the kdcluster CRD.

Reply:
I changed this.